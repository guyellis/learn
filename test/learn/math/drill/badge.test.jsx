
const React = require('react');
const renderer = require('react-test-renderer');
const MuiThemeProvider = require('material-ui/styles/MuiThemeProvider').default;
const getMuiTheme = require('material-ui/styles/getMuiTheme').default;
const lightBaseTheme = require('material-ui/styles/baseThemes/lightBaseTheme').default;
const Badge = require('../../../../src/learn/math/drill/badge');

const muiTheme = getMuiTheme(lightBaseTheme);

test('Badge should be rendered', () => {
  const component = renderer.create(
    <MuiThemeProvider muiTheme={muiTheme}>
      <Badge
        color={0}
        content="+"
      />
    </MuiThemeProvider>);
  const badgeTotals = component.toJSON();
  expect(badgeTotals).toMatchSnapshot();
});
