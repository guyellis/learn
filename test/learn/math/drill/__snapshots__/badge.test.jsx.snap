// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge should be rendered 1`] = `
<div
  aria-describedby={null}
  className="MuiAvatar-root MuiAvatar-colorDefault"
  onBlur={[Function]}
  onFocus={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
  onTouchEnd={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "backgroundColor": "#fbc02d",
    }
  }
  title="+ Gold"
>
  +
</div>
`;
